integrations/Posthog/browser.js

```diff
@@ -55,29 +55,24 @@ class Posthog {
         (e._i = []),
         (e.init = function (i, s, a) {
           function g(t, e) {
-            const o = e.split(".");
```

Comment:
Multiple syntax differences across this file. Please take look.

Reply:
You mean w.r.t quotes around strings?